### PR TITLE
feat: add initial action

### PR DESCRIPTION
**Pull request by sebastienrobidoux**
FIXES: #PF-5867

[Link to PF-5867](https://jira.croesus.com/browse/PF-5867)

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          echo "cd ${{ inputs.version }}";
+          echo "mkdir rpm";
+          echo "cd rpm";
+          echo "put ./new-rpm/*";
+          echo "ls -l";
+          echo "exit"
+        ) > sftp-command.txt
+      shell: bash
+
+    - name: Transfert to Devarch
```
Comment (vladolto) [inline action.yaml:58]: Pas de t en anglais :)

Comment (sebastienrobidoux) [reply action.yaml]: Fixed!

File: action.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      # Use SFTP using the command file generate on the previous step 
+      # builddocker MUST have the credentials to connect to ${{ input.devarch-user }}@devarch
+      # Assume that the folder /data/archives/${{ input.project-name }} already exist
+      # The upload will fail otherwise
+      run: sftp -b sftp-command.txt ${{ input.devarch-user }}@devarch:/data/archives/${{ input.project-name }}
+      shell: bash
```
[the remark this answers is not in the document]
Comment (sebastienrobidoux) [reply action.yaml]: Fixed!

**Comment by sebastienrobidoux** [comment]
Test in progress
https://docs.google.com/spreadsheets/d/1I-KLYr0Q-sVHJzvjsVJjUx39G9DXRikuDjV6oPY56ew/edit#gid=0

**Pull request merged**